proto-lens-protoc/src/Data/ProtoLens/Compiler/Generate/Field.hs

```diff
@@ -6,17 +6,68 @@
 --
 -- Upstream docs:
 -- <https://developers.google.com/protocol-buffers/docs/encoding#structure>
-module Data.ProtoLens.Compiler.Generate.FieldEncoding
-    ( FieldEncoding(..)
+module Data.ProtoLens.Compiler.Generate.Field
+    ( hsFieldType
+    , hsFieldVectorType
+    , FieldEncoding(..)
     , fieldEncoding
     , lengthy
     , groupEnd
     , isolatedLengthy
     ) where
 
+import Data.Text (unpack)
```

Comment:
Do you have a particular style you follow for imports sorting?

Reply:
Reordered.  Usually I do a sorted block of imports from other packages, followed by a block from the current package.  But I'm sure I haven't been completely consistent.